sourcecode/apis/contentauthor/app/ApiModels/User.php

```diff
@@ -22,18 +22,18 @@ public function getId(): string
         return $this->id;
     }
 
-    public function getFirstName(): string
+    public function getFirstName($default = null): ?string
```

Comment:
Having a `$default` parameter isn't very useful when it's restricted by the return type, and `??` is so easy to use anyway. I suggest removing.

Reply:
Agreed. I have changed to `??` instead